.drone.star

```diff
@@ -1183,9 +1192,10 @@ def localApiTests(ctx, name, suites, storage = "ocis", extra_environment = {}, w
         "BEHAT_FILTER_TAGS": "~@skip&&~@skipOnGraph&&~@skipOnOcis-%s-Storage" % ("OC" if storage == "owncloud" else "OCIS"),
         "EXPECTED_FAILURES_FILE": expected_failures_file,
         "UPLOAD_DELETE_WAIT_TIME": "1" if storage == "owncloud" else 0,
-        "OCIS_WRAPPER_URL": "http://%s:5200" % OCIS_SERVER_NAME,
+        "OCIS_WRAPPER_URL": "http://ocis-pods:5200" if k3d else "http://%s:5200" % OCIS_SERVER_NAME,
         "WITH_REMOTE_PHP": with_remote_php,
         "COLLABORATION_SERVICE_URL": "http://wopi-fakeoffice:9300",
+        "K3D": "true" if k3d else "false",
```

Comment:
k3d is just a tool that we use in CI. but we should be able to run tests against any k8s cluster
```suggestion
        "K8S": "true" if k3d else "false",
```